File: Python_v_C/factorial.c
/*
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
|r|e|d|a|n|d|g|r|e|e|n|.|c|o|.|u|k|
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
*/
// Calculate the factorial in C

#include <stdio.h>

int main()
{

    int num;
    int fact;
    int i;
    printf("Enter a number\n");
    scanf("%d", &num);
    fact = 1;
    for (i = 1; i < num; i++)
    {
        fact *= i;
    }
    printf("%d", fact);
}

File: Python_v_C/factorial.py

# +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
# |r|e|d|a|n|d|g|r|e|e|n|.|c|o|.|u|k|
# +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+

# Calculate the factorial in Python

num = input("Enter a number \n")

num = int(num)
fact = 1

for i in range(1, num+1):
    fact *= i

print(fact)
